sources/packages/backend/apps/api/src/route-controllers/student/_tests_/e2e/student.institutions.controller.searchStudents.e2e-spec.ts

```diff
@@ -0,0 +1,402 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import { DataSource, Repository } from "typeorm";
+import {
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getInstitutionToken,
+  getAuthRelatedEntities,
+  InstitutionTokenTypes,
+  authorizeUserTokenForLocation,
+} from "../../../../testHelpers";
+import {
+  createFakeInstitutionLocation,
+  saveFakeApplication,
+  saveFakeStudent,
+} from "@sims/test-utils";
+import {
+  Application,
+  ApplicationStatus,
+  Institution,
+  InstitutionLocation,
+  Student,
+} from "@sims/sims-db";
+
+describe("StudentInstitutionsController(e2e)-searchStudents", () => {
+  let app: INestApplication;
+  let appDataSource: DataSource;
+  let collegeF: Institution;
+  let collegeFLocation: InstitutionLocation;
+  let studentRepo: Repository<Student>;
+  let applicationRepo: Repository<Application>;
+  let collegeFInstitutionUserToken;
```

Comment:
Please type this to avoid VSCode waning.
![image](https://user-images.githubusercontent.com/61259237/232627700-4bc72372-8437-4c47-8253-0133ab23d55a.png)